src/structures/GuildScheduledEvent.js

```diff
@@ -303,6 +303,7 @@ class GuildScheduledEvent extends Base {
 
   /**
    * Sets the new status of the guild scheduled event.
+   * <info>Use this method in conjunction with status type-guards like {@link GuildScheduledEvent.isScheduled}</info>
```

Comment:
Make sure this says "if you're using TypeScript", otherwise we'll get people confused on why they need to use this

Reply:
`{@link GuildScheduledEvent.isScheduled}` should be `{@link GuildScheduledEvent#isScheduled}`.